src/test/java/com/ncc/aif/NistTA3ExamplesAndValidationTest.java

```diff
@@ -303,9 +306,17 @@ void invalid() {
             }
 
             @Test
-            void valid() {
+            void validWithoutCluster() {
                 utils.makeValidTA3Hypothesis(entity, relation, relationEdge, event, eventEdge);
-                utils.testValid("NISTHypothesis.valid: All KEs in model must be referenced by hypothesis");
+                utils.testValid("NISTHypothesis.validWithoutClusters: All KEs in model must be referenced by hypothesis");
+            }
+
+            @Test
+            void validWithClusterAndMembership() {
+                ResIterator it = model.listSubjectsWithProperty(AidaAnnotationOntology.CLUSTER_PROPERTY, relationCluster);
+                Assertions.assertTrue(it.hasNext(), "Unable to find expected cluster membership");
+                utils.makeValidTA3Hypothesis(entity, relation, relationEdge, event, eventEdge, relationCluster, it.nextResource());
```

Comment:
This generated AIF that I wouldn't have expected.  It put a blank node in the hypothesis subgraph (attached).  I think the ClusterMemberships should be named.

[KEsInModelMustBeReferencedByHypothesis.validWithClusterAndMembership.ttl.txt](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/files/3239038/KEsInModelMustBeReferencedByHypothesis.validWithClusterAndMembership.ttl.txt)

Reply:
ClusterMembership nodes have always been blank. Are you suggesting that this ticket be used to make a change like naming ClusterMemberships?

Reply:
Per discussion, have disabled these tests until KE restriction is implemented correclty